application-analytics-ui/src/main/resources/Analytics/Code/Configuration.xml

```diff
@@ -90,40 +104,41 @@
         <name>trackingCode</name>
         <number>1</number>
         <picker>1</picker>
-        <prettyName>Matomo Tracking Code</prettyName>
+        <prettyName>Tracking Code</prettyName>
         <rows>20</rows>
         <size>40</size>
         <unmodifiable>0</unmodifiable>
         <validationMessage/>
         <validationRegExp/>
         <classType>com.xpn.xwiki.objects.classes.TextAreaClass</classType>
       </trackingCode>
-      <requestAddress>
-        <customDisplay/>
-        <disabled>0</disabled>
-        <hint/>
-        <name>requestAddress</name>
-        <number>4</number>
-        <picker>1</picker>
-        <prettyName>requestAddress</prettyName>
-        <size>30</size>
-        <unmodifiable>0</unmodifiable>
-        <validationMessage/>
-        <validationRegExp/>
-        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
-      </requestAddress>
     </class>
     <property>
-      <authToken/>
+      <authToken>d20b0be0807053ea1fc03705483b9f2a</authToken>
     </property>
     <property>
-      <siteId/>
+      <requestAddress>http://130.61.233.19/matomo/</requestAddress>
     </property>
     <property>
-      <trackingCode/>
+      <siteId>3</siteId>
     </property>
     <property>
-      <requestAddress/>
+      <trackingCode>&lt;!-- Matomo --&gt;
+&lt;script&gt;
+  var _paq = window._paq = window._paq || [];
+  /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+  _paq.push(['trackPageView']);
+  _paq.push(['enableLinkTracking']);
```

Comment:
Changes to this page added by mistake. Probably for the ones above as well? Please make sure to review the added changes with `git add -p`